client/components/gsuite/gsuite-new-user-list/new-user.tsx

```diff
@@ -128,12 +121,10 @@ const GSuiteNewUser: FunctionComponent< Props > = ( {
 	};
 
 	return (
-		<div
-			className={ classNames( 'gsuite-new-user-list__new-user', { 'show-labels': showLabels } ) }
-		>
+		<div className={ classNames( 'gsuite-new-user-list__new-user' ) }>
```

Comment:
We can't remove the `show-labels` class without updating the style that is dependent on the class:
https://github.com/Automattic/wp-calypso/blob/update/labels-in-add-new-mailboxes-for-google-workspace/client/components/gsuite/gsuite-new-user-list/style.scss#L66